hrjobcontract/CRM/Hrjobcontract/Upgrader.php

```diff
@@ -513,6 +513,7 @@ public function upgradeBundle() {
     $this->upgrade_1032();
     $this->upgrade_1033();
     $this->upgrade_1036();
```

Comment:
I noticed calls to 1034 and 1035 are missing from `upgradeBundle`. It's not critical that these are run during installation (they will be run later anyway) but can you add them in this PR just to be consistent?